src/main/xerus/monstercat/downloader/TabDownloader.kt

```diff
@@ -347,7 +347,7 @@ class TabDownloader: VTab() {
 				show()
 			}
 		}, Button("Checking connection...").apply {
-			prefWidth = 200.0
+			prefWidth = Control.USE_COMPUTED_SIZE
```

Comment:
The button shouldn't change in size when you interact with it. Why did you change this? Also, if it should use the computed size you can simply remove the line ^^

Reply:
I use Computed Size because the text inside the button changes and 200 wasn't enough. I find it better to have the button exactly to it's text size.

I could simply make prefWidth higher.

Reply:
Then do that. In my experience it looks really weird if the button resizes automatically, and it can lead to annoying interaction issues.